graphql/database/cause/causes/cats/impact.referralRewardTitle.md

```diff
@@ -1 +1 @@
-##### **You just put \${pendingUserReferralImpact} cat\${isPlural(referralImpact)}** on track for adoption!
+#### You just put \${claimedReferralImpact} cats on track for adoption!
```

Comment:
Is the lack of pluralization an issue? Probably fine for now with the fixed 10 cats?

Reply:
figured it was fine for now b/c we always give plural referral bonus.  Made it similar to seas